docs/data/additional_components.yml

```diff
@@ -21,11 +21,11 @@ flink-connector-parent:
   source_release_asc_url: "https://downloads.apache.org/flink/flink-connector-parent-1.0.0/flink-connector-parent-1.0.0-src.tgz.asc"
   source_release_sha512_url: "https://downloads.apache.org/flink/flink-connector-parent-1.0.0/flink-connector-parent-1.0.0-src.tgz.sha512"
 
-flink-shaded-17.0:
-  name: "Apache Flink-shaded 17.0 Source Release"
-  source_release_url: "https://www.apache.org/dyn/closer.lua/flink/flink-shaded-17.0/flink-shaded-17.0-src.tgz"
-  source_release_asc_url: "https://downloads.apache.org/flink/flink-shaded-17.0/flink-shaded-17.0-src.tgz.asc"
-  source_release_sha512_url: "https://downloads.apache.org/flink/flink-shaded-17.0/flink-shaded-17.0-src.tgz.sha512"
+flink-shaded-18.0:
+  name: "Apache Flink-shaded 18.0 Source Release"
+  source_release_url: "https://www.apache.org/dyn/closer.lua/flink/flink-shaded-18.0/flink-shaded-18.0-src.tgz"
+  source_release_asc_url: "https://downloads.apache.org/flink/flink-shaded-18.0/flink-shaded-18.0-src.tgz.asc"
+  source_release_sha512_url: "https://downloads.apache.org/flink/flink-shaded-18.0/flink-shaded-18.0-src.tgz.sha512"
 
 flink-shaded-16.2:
```

Comment:
Keep `18.0` and `16.2` isn't a good idea here.

Should we keep the 17.0 and 18.0? And remove the 16.2? Or keep all versions here?

Reply:
That's actually a good question
the only thing making me stopping before removal that is that it was released just a couple of weeks ago...
However I think you are right and  probably we should not stick to it

Reply:
I see the `pre-bundled-hadoop-` has 4 versions in this yml, so should we keep 16, 17 and 18?

Reply:
+1, maybe it is a good choice to keep 16.2, 17.0, and 18.0.

Reply:
thanks for the feedback, I added 17.0 back to the site